homeassistant/components/nanoleaf/light.py

```diff
@@ -210,7 +210,15 @@ def turn_on(self, **kwargs):
                 self._light.brightness = int(brightness / 2.55)
 
         if effect:
-            self._light.effect = effect
+            # Nanoleaf http api returns non-existent effect named "*Solid*" when light set to solid color.
```

Comment:
The check here is always good to have since the user can call the service manually with incorrect effects. So just add the comment in the update method, not here.

Reply:
Done